edge-22.10.1 change notes (#9553)
linkerd/linkerd2: CHANGES.md

# Changes

## edge-22.10.1

> This edge release fixes some sections of the Viz dashboard that were appearing

Reviewer: i think that we can drop the "that were" here to make this a little more concise?
```suggestion
This edge release fixes some sections of the Viz dashboard appearing
```

> blank, and adds an optional PodMonitor resource to the Helm chart which allows
> having an easier integration with the Prometheus Operator. It also includes many

Reviewer: grammar nit: "allows having an easier integration" feels kind of awkward to me, maybe consider something like:
```suggestion
blank, and adds an optional PodMonitor resource to the Helm chart to enable
easier integration with the Prometheus Operator. It also includes many
```

fixes submitted by our contributors.

> * Fixed the dashboard sections Tap, Top, Routes that were appearing blank
>   (thanks @MoSattler!)

Reviewer: nit, maybe
```suggestion
* Fixed the web dashboard sections Tap, Top, and Routes appearing blank
  (thanks @MoSattler!)
```

* Added an optional PodMonitor resource to the main Helm chart (thanks
  @jaygridley!)

> * Fixed the CLI's `--api-addr` flag, that was being ignored (thanks @mikutas!)

Reviewer: this might be clearer as just:
```suggestion
* Fixed the CLI ignoring the `--api-addr` flag (thanks @mikutas!)
```

> * Fixed the CLI's `--api-addr` flag (thanks @mikutas!)

Reviewer: what did we fix about the `--api-addr` flag?

Author: It was being ignored. I've expanded the notes to explain :+1:

* Expanded the `linkerd authz` command to display AuthorizationPolicy resources
  that target namespaces (thanks @aatarasoff!)
* Fixed interpretation of the `NotIn` label selector operator in the policy

>   resources

Reviewer: take it or leave it: "fixed interpretation"  feels a little vague to me, maybe we should note specifically that it was essentially being treated as an `In` operator previously?

> * Fixed the `NotIn` label selector operator in the policy resources, that was
>   erroneously treated as `In`.

Reviewer: how about:
```suggestion
* Fixed the `NotIn` label selector operator in the policy resources being
  erroneously treated as `In`.
```
